src/lang/db.pl

```
@@ -69,13 +69,20 @@
 
 %%
 mng_import(Dir) :-
-	forall(
-		(	collection_name(Name),
-			mng_get_db(DB, Collection, Name)
+	findall((DB, Dir0),
+		(	
+			collection_name(Name),
+			mng_get_db(DB, Collection, Name),
+			path_concat(Dir, Collection, Dir0),
+			exists_directory(Dir0)
 		),
-		(	path_concat(Dir, Collection, Dir0),
-			mng_restore(DB, Dir0)
-		)
+		DirCollection
+	),
+	% Fails if there is no directory to import
+	not(length(DirCollection, 0)),
```

this should not silently fail, a warning should be logged